app/src/main/java/com/lyrics/feelin/presentation/view/onboarding/AGENTS.md

```
@@ -0,0 +1,42 @@
+```text
+onboarding/
+├── OnboardingViewModel.kt
+├── favoriteartist/
+├── genderage/
+├── profile/
+├── terms/
+└── welcome/
+```
+
+## WHERE TO LOOK
+| Task | Location | Notes |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)